gaia-rs/src/store_keys.rs

```diff
@@ -16,6 +16,8 @@ pub enum GaiaStoreKey {
     IBC,
     #[skey(to_string = "capability")]
     Capability,
+    #[skey(to_string = "genutil")]
+    Genutil,
```

Comment:
Is `app_state` contains all StoreKeys or it smth different?

Reply:
From what I can see in Gaia genutils doesn't have an entry in the store

Reply:
But it has to be in genesis. Okay, thanks.

Reply:
Removed